pkg/scheduler/plugins/predicates/predicates.go

```diff
@@ -109,9 +119,15 @@ type predicateEnable struct {
 	podTopologySpreadEnable bool
 	cacheEnable             bool
 	proportionalEnable      bool
+	volumeBindingEnable     bool
 	proportional            map[v1.ResourceName]baseResource
 }
 
+// bind context extension information of predicates
+type bindContextExtension struct {
```

Comment:
The `skipPlugins` var in predicates is not so necessary after each task has a cycle state, we can refactor it.

Reply:
I have refactor the `handleSkipPrePredicatePlugin` and `handleSkipPredicatePlugin`, and current log info in them are wrong, I also have changed it, log in handleSkipPrePredicatePlugin is enough